boxes.yaml.example

```diff
@@ -38,15 +38,26 @@ katello-client:
 centos7-bats-ci:
```

Comment:
It might be nicer to rename this to centos7-katello-bats-ci

Reply:
Note this will require a CI change -- https://github.com/theforeman/foreman-infra/pull/333